packages/odyssey-react/src/components/Box/_box-root.scss

```diff
@@ -21,4 +21,158 @@
   line-height: $base-line-height;
   font-feature-settings: "lnum", "pnum";
   text-decoration-skip-ink: all;
+
+  // Catch-all border-box bad boy
+  /* stylelint-disable selector-max-universal */
+  * {
+    box-sizing: border-box;
+
+    &::before,
+    &::after {
+      box-sizing: inherit;
+    }
+  }
+  /* stylelint-enable selector-max-universal */
+
+  // Huge selector list layout reset
+  /* stylelint-disable selector-max-type */
+  div,
+  span,
+  applet,
+  object,
+  iframe,
+  h1,
+  h2,
+  h3,
+  h4,
+  h5,
+  h6,
+  p,
+  blockquote,
+  pre,
+  a,
+  abbr,
+  acronym,
+  address,
+  big,
+  cite,
+  code,
+  del,
+  dfn,
+  em,
+  img,
+  ins,
+  kbd,
+  q,
+  s,
+  samp,
+  small,
+  strike,
+  strong,
+  sub,
+  sup,
+  tt,
+  var,
+  b,
+  u,
+  i,
+  center,
+  dl,
+  dt,
+  dd,
+  ol,
+  ul,
+  li,
+  fieldset,
+  form,
+  label,
+  legend,
+  table,
+  caption,
+  tbody,
+  tfoot,
+  thead,
+  tr,
+  th,
+  td,
+  article,
+  aside,
+  canvas,
+  details,
+  embed,
+  figure,
+  figcaption,
+  footer,
+  header,
+  hgroup,
+  menu,
+  nav,
+  output,
+  ruby,
+  section,
+  summary,
+  time,
+  mark,
+  audio,
+  video {
+    margin-block: 0;
+    margin-inline: 0;
+    padding-block: 0;
+    padding-inline: 0;
+    border: 0;
+    font: inherit;
+    font-size: 100%;
+    vertical-align: baseline;
+  }
+
+  button,
+  input {
+    font: inherit;
+    font-size: 100%;
+  }
+
+  // HTML5 display-role reset for older browsers
+
+  article,
+  aside,
+  details,
+  figcaption,
+  figure,
+  footer,
+  header,
+  hgroup,
+  menu,
+  nav,
+  section {
+    display: block;
+  }
+
+  // Element Specific Resets
+
+  img,
+  svg {
+    max-width: 100%;
+  }
+
+  ol,
+  ul {
+    list-style: none;
+  }
+
+  blockquote,
```

Comment:
How do we decide which styles are included here and which are included in Text?

Reply:
I don't know.

The proposed path forward is to wrap every component in a `Box` as its root and remove the global style sheet. 

If any of the children of that component were relying on the global element styles and these are removed then they break, but it's difficult to say where that will happen.

At that point each of those broken elements need a wrapper of some type, `Text` or `Box` depending.

If we leave these here the content making those assumptions "just works" kind of. The caveat being now those element selectors that previously had a specificity of 1 have been bumped to 2, and causes another set unexpected overrides.